packages/server/src/application/types/index.ts

```diff
@@ -337,8 +337,19 @@ export type SocketService = {
   identify: (stream: SocketStream | Socket) => Auth['apiTokenPayload'] | null;
 };
 
+export interface TestRequest {
+  get: (url: string) => this;
+  post: (url: string) => this;
+  authorize: (user: User) => this;
+  setCookies: (cookies: string[]) => this;
+  send: (data: Object) => this;
+  run: () => Promise<any>;
+  reset: () => void;
+}
+
```

Comment:
Me being nit-picky - Can we update method names please? 

get :heavy_check_mark: 
post :heavy_check_mark: 
authorize :warning: withAuthorization?
setCookies :warning:  withCookies?
send :no_entry: withBody?
run :runner: I know I suggested it but was hoping you would come up with something better or call me out :smile: